tuftool/tests/create_repository_integration.rs

```diff
@@ -0,0 +1,268 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT OR Apache-2.0
+
+mod test_utils;
+use assert_cmd::Command;
+use chrono::{Duration, Utc};
+use std::fs::File;
+use tempfile::TempDir;
+use tough::{ExpirationEnforcement, Limits, Repository, Settings};
+
+// This file include integration tests for KeySources: tough-ssm, tough-kms and local file key.
+// Since the tests are run using the actual "AWS SSM and AWS KMS", you would have to configure
+// AWS account profile "test-profile" with root permission.
+// Refer https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-profiles.html to configure named profile.
+// Additionally, tough-kms key generation is not supported (issue  #211), so you would have to manually create kms CMK key.
+// To run test include feature flag 'integration-tests' like : "cargo test --features=integration-tests"
+
+const PROFILE: &str = "test-profile";
+
+fn initialize_root_json(root_json: &str) {
+    Command::cargo_bin("tuftool")
+        .unwrap()
+        .args(&["root", "init", root_json.clone()])
+        .assert()
+        .success();
+    Command::cargo_bin("tuftool")
+        .unwrap()
+        .args(&["root", "expire", root_json.clone(), "2020-09-22T00:00:00Z"])
```

Comment:
Seems like this will make tests start failing in a month... :)

```suggestion
        .args(&["root", "expire", root_json.clone(), "3030-09-22T00:00:00Z"])
```

Reply:
Oops. Done.